modules/metastorage/src/test/java/org/apache/ignite/internal/metastorage/server/AbstractCompactionKeyValueStorageTest.java

```diff
@@ -157,7 +157,7 @@ public void tearDown() throws Exception {
     void testCompactRevision1() {
         storage.compact(1);
 
-        assertEquals(List.of(3, 5), collectRevisions(FOO_KEY));
+        assertEquals(List.of(1, 3, 5), collectRevisions(FOO_KEY));
```

Comment:
`compact` method's javadoc says:

```
@param revision Revision up to which (including) the metastorage keys will be compacted
```

Why do we have the `1` revision here? And why doesn't `BAR_KEY` have this revision?

Reply:
Sadly, that javadoc doesn't really define what it means by "compacted", and that's a source of great confusion among many people now. I will improve that javadoc